apps/daimo-mobile/src/view/screen/receive/DepositScreen.tsx

```diff
@@ -67,34 +66,25 @@ function DepositScreenInner({ account }: { account: Account }) {
 function OnrampsSection({ account }: { account: Account }) {
   const [started, setStarted] = useState(false);
 
-  const openCBPay = useCallback(() => {
-    const cbUrl = generateOnRampURL({
-      appId: "2be3ccd9-6ee4-4dba-aba8-d4b458fe476d",
-      destinationWallets: [
-        {
-          address: account.address,
-          blockchains: ["base"],
-          assets: ["USDC"],
-          supportedNetworks: ["base"],
-        },
-      ],
-      handlingRequestedUrls: true,
-      defaultExperience: "send",
-    });
-
-    Linking.openURL(cbUrl);
+  const openRecommendedExchange = (url: string) => {
+    Linking.openURL(url);
     setStarted(true);
-  }, [account]);
+  };
 
   return (
     <View>
       <HeaderRow title="Recommended exchanges" />
 
-      <ButtonMed
-        type="primary"
-        title="Deposit from Coinbase"
-        onPress={openCBPay}
-      />
+      {account.recommendedExchanges.map((rec, idx) => (
+        <View key={idx}>
+          <ButtonMed
+            type="primary"
+            title={rec.cta}
+            onPress={() => openRecommendedExchange(rec.url)}
+          />
+          {idx < account.recommendedExchanges.length - 1 && <Spacer h={16} />}
```

Comment:
might make only the first one primary and rest `subtle`

Reply:
this is current:

<img width="256" src="https://github.com/daimo-eth/daimo/assets/6984346/99adc075-2e03-437b-951e-018ce3d2600d">
<img width="256" src="https://github.com/daimo-eth/daimo/assets/6984346/042a3c8f-d5a3-47bc-85a5-84483895acab">